Orange/data/sql/mysql.py

```diff
@@ -0,0 +1,288 @@
+import logging
+from contextlib import contextmanager
+
+from Orange.util import Registry
+import mysql.connector as mysql
```

Comment:
You also have to add the requirement to https://github.com/biolab/orange3/blob/master/requirements-sql.txt

Reply:
Done. Still pending tests!